### DISCUSSION TITLE
Add MCU Temp monitoring and get crashes.

### DISCUSSION THREAD

**Discussion by kr0n1k01** [bassamanator/everything-sovol-sv06#26] [Q&A]
So when building out the config for the sv06+ I chose to copy a few things from my Ender 3v2 and found a couple quirks. 

First when adding (mcu below) 
---------------------------------
[temperature_sensor mcu_temp]
sensor_type: temperature_mcu
min_temp: 0
max_temp: 110
---------------------------------
for some reason this causes the machine to crash when probing the z axis and going under 0 (coincidence possibly/probably however 100% repeatable) causes a  - MCU shutdown: ADC out of range - when reviewing the logs shows my MCU temp spiked to -409 and causes a crash. Is there another command I can use or is there a underlying problem here? Removing the monitoring does resolve this and allow to function without issue. 

Second max extruder temp I had to bump this to 310 in the config as during my testing I set the extruder temp to 300 to verify device would reach it. Again got a system crash this time because during heatup the temp exceeded 300 and sensitve klipper said no no no and shut down.  

For the experienced out there is there any known issue to exceeding to the limits in of the device in the config to avoid things like this? 

Aside from the 2 things mentioned above everything went super smooth and am entirely grateful for this extremely simple thorough guide.

**Comment by bassamanator** [comment]
`[temperature_sensor mcu_temp]` is not included in my config because of what you experienced. I use this section on my Voron 0.1 issue free, however, with this board, this section is a no-no. If you could find a way to 'delay' this section from kicking in, you will have solved this issue. Let me know how it goes.

The second one is not an issue, the behaviour is normal. If you are in fact going to be printing at 300C, then you will absolutely have to set max temp to around 310C, there is no way around it. If you're not going to be printing that hot, then I suggest you set max temp to 300C, and leave it there.

Glad you found my work useful 😀. Please star the repo if possible. Thanks!

And please post a solution if you find one for the mcu temp.

**Reply by kr0n1k01** [reply]
Well that explains it perfectly.  I do wonder what voodoo they did on the board to break this.

GTK - Klipper is def super duper sensitive to keeping you within the limits as to what is defined.


Thanks again for the work u have done and the response lets me know I am not out of my mind completely.
